src/components/views/messages/TextualBody.js

```diff
@@ -128,6 +132,13 @@ export default class TextualBody extends React.Component {
         }
     }
 
+    _addCodeElement(pre) {
+        const code = document.createElement("code");
+        code.append(...pre.childNodes);
+        pre.innerHTML = "";
```

Comment:
Is this actually required...? Appending existing nodes to a different place typically _moves_ them with most DOM APIs, so I would expect `pre` to already be empty...

Reply:
Yeah, you're right. Sorry for all the trouble with this...

Reply:
No worries, no need to stress about it. 😄